src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -212,8 +213,17 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     /// Gets or sets the content to render when <see cref="Loading"/> is true.
     /// A default fragment is used if loading content is not specified.
     /// </summary>
-    [Parameter]
-    public RenderFragment? LoadingContent { get; set; }
+    [Parameter] public RenderFragment? LoadingContent { get; set; }
+
+    /// <summary>
+    /// Sets <see cref="GridTemplateColumns"/> to automatically fit the columns to the available width as best it can.
+    /// </summary>
+    [Parameter] public bool AutoFit { get; set; }
+
+    /// <summary>
+    /// Gets the first (optional) SelectColumn
+    /// </summary>
+    internal IEnumerable<SelectColumn<TGridItem>> SelectColumns => _columns.Where(col => col is SelectColumn<TGridItem>).Cast<SelectColumn<TGridItem>>();
```

Comment:
Why is this here? I don't see it used in your code. Please remove if you are not using it.